cime_config/config_pes.xml

```diff
@@ -187,7 +187,7 @@
       </pes>
     </mach>
   </grid>
-  <grid name="a%0.9x1.25.+oi%tnx1v4">
+  <grid name="a%0.9x1.25.+oi%tnx1v4|a%1.9x2.5.+oi%tnx1v4">
```

Comment:
```suggestion
  <grid name="a%0.9x1.25.+oi%tnx1v4 | a%1.9x2.5.+oi%tnx1v4">
```
Question: Will it also work if you include spaces around the pipe symbol? Maybe improves readability.

Reply:
Not sure, there is no space other places, so I'd suggest to leave it for now.